types/diff-match-patch/tslint.json

```diff
@@ -13,6 +13,8 @@
                 ],
                 "mode": "code"
             }
-        ]
+        ],
+        "no-const-enum": false,
+        "unified-signatures": false
```

Comment:
Please use this inline (`// tslint:disable:....`) as per README, unless there is a large number of affected lines